inst/schema/ValidateInputResponse.schema.json

```diff
@@ -1,3 +1,52 @@
 {
-	"type": "string"
+  "definitions": {
+    "input_response": {
+      "properties": {
+        "filename": { "$ref": "FileName.schema.json" },
+        "type": {
+          "type": "string"
+        },
+        "data": {
+          "type": "object"
+        }
+      },
+      "additionalProperties": false,
+      "required": [ "filename", "type", "data" ]
+    },
+
+    "pjnz_response": {
+      "allOf": [
+        { "$ref": "#/definitions/input_response" },
+        { "properties": 
+          {
+            "filename": {},
+            "type": { 
+              "const": "pjnz"
```

Comment:
Do you think we could change these `const` values to enums with single values instead? It's just that I haven't found a typescript package that supports draft 6 yet :( 

According to the spec "const is merely syntactic sugar for an enum with a single element" https://json-schema.org/understanding-json-schema/reference/generic.html#constant-values

Reply:
Can you not use `ajv` from typescript? That's what we're using and it looks like it has typescript support?

Reply:
This isn't for validation, it's for actually generating typescript types/interfaces from the schema. I'm using https://www.npmjs.com/package/json-schema-to-typescript.

The only other one I've found is https://github.com/quicktype/quicktype, I can try with that one.

Reply:
Actually I was inferring from this issue that the latter will not support `const` either: https://github.com/quicktype/quicktype/issues/1225

Reply:
Sorry, hadn't realised this was for generating things. Rob can easily change this over to being a single-value enum (we talked about this already in person)

Reply:
That'd be great, thanks :+1:

Reply:
Sorry for the necro, but did you eventually find some library that deals with `"const"` properly for code generation?